frameworks/digital-outcomes-and-specialists/messages/urls.yml

```diff
@@ -1 +1,5 @@
+call_off_contract_url: "https://www.gov.uk/government/publications/digital-outcomes-and-specialists-call-off-contract"
+
+framework_agreement_url: "https://www.gov.uk/government/publications/digital-outcomes-and-specialists-framework-agreement"
```

Comment:
Did you just add this one to be neat and tidy? Rather than it being a requirement for the story?